static-parts/README-start.md

```diff
@@ -9,6 +9,7 @@
 - Use <kbd>Ctrl</kbd> + <kbd>F</kbd> or <kbd>command</kbd> + <kbd>F</kbd> to search for a snippet.
 - Contributions welcome, please read the [contribution guide](CONTRIBUTING.md).
 - Snippets are written in ES6, use the [Babel transpiler](https://babeljs.io/) to ensure backwards-compatibility.
+- You can import these snippets into your text editor of choice (VSCode, Atom, Sublime) using the files found in [this repo](https://github.com/Rob-Rychs/30-seconds-of-code-texteditorsnippets).
```

Comment:
Is this the correct repo link as you're generating and hosting the text editor snippets?

Reply:
Hi @skatcat31, yes that is correct. 

It's a goal to automate the generation/find some clever way to keep them in sync with Chalarangelo/30-seconds-of-code , but for now i'll just watch for changes and update the files periodically (~once a month).

Could probably package-up *30-seconds-of-code* text editor extensions from these files down the road...

Reply:
Possibly something with a hook and CI? Or maybe a weekly run script on CI?